README.md

```diff
@@ -117,42 +117,111 @@ dependencies {
 ### Configuration
 
 In order to configure the Splunk RUM library, you will need to know three things:
-* Your Splunk realm. 
+
+* Your Splunk realm.
   * The realm can be found in your Splunk Observability UI in the Account Settings page.
-* Your RUM access token.  
-  * You can find or create a RUM access token in the Splunk Observability UI, in your Organization Settings.
-  * Important: this access token *must* have the `RUM` authorization scope to work. 
+* Your RUM access token.
+  * You can find or create a RUM access token in the Splunk Observability UI, in your Organization
+    Settings.
+  * Important: this access token *must* have the `RUM` authorization scope to work.
 * The name of your application.
 
 Here is an example of a the very minimal configuration which uses these 3 values:
+
 ```java
-        String beaconUrl = "<realm>";
-        String rumAccessToken = "<your_RUM_access_token>";
-        Config config = SplunkRum.newConfigBuilder()
-                .realm(realm)
-                .rumAccessToken(rumAuth)
-                .applicationName("My Android App")
-                .build();
+  String beaconUrl="<realm>";
+  String rumAccessToken="<your_RUM_access_token>";
+  Config config=SplunkRum.newConfigBuilder()
```

Comment:
Nit: spaces around `=` looked a bit nicer

Reply:
yeah, Android Studio keeps trying to format the markdown and messing it up. 😢

Reply:
I think I'm going to put a full java class in here to see if it does a better job not messing it up.

Reply:
done, and IDEA is much happier formatting it now